### DISCUSSION TITLE
Linux build question

### DISCUSSION THREAD

**Discussion by Jacobw1oo** [Heroic-Games-Launcher/HeroicGamesLauncher#2653] [Q&A] [answered]
Command: 
yarn dist:linux # Optionally specify a package to create (eg: deb, pacman, tar.xz, rpm, AppImage); default: AppImage

How do i specify the package to create for  pacman?

Extra: i am building on Aarch64 arch Linux (gnome - Wayland)

**Comment by imLinguin** [comment, marked as answer]
That would be `yarn dist:linux pacman`

**Reply by Jacobw1oo** [reply]
Thanks

**Comment by Jacobw1oo** [comment]
Update: for Aarch64 its: USE_SYSTEM_FPM=TRUE yarn dist:linux pacman

Due to Electron building downloading fpm x86_64 and trying to use it, it also errors out.
